script/test

```diff
@@ -4,6 +4,12 @@
 
 set -e
 
+echo "==> Checking shell scripts"
+
+shellcheck ./script/**/*[^utils] ./script/utils/**
```

Comment:
Does the npm package make `shellcheck` available globally then? I'd have thought we'd have to run it as an npm command?

Reply:
Given the description in [the usage section of the README](https://www.npmjs.com/package/shellcheck) I think it just makes it available globally but I've added the path to the binary just to be sure